Add cloud engineering summit as listing on events and webinar (#3942)
pulumi/docs: content/events/cloud-engineering-summit/index.md

---
# Name of the event.
title: "Cloud Engineering Summit"

# Events with unlisted as true will not be shown on the event list
unlisted: false

# Events with external registrations should not be indexed
# and have redirect to the external registration page.
block_external_search_index: true

> redirect_to: "https://cloudengineering.heysummit.com/"

Reviewer: Not part of this PR, but as a general rule, we should probably tack on a `rel="noopener noreferrer"` to any links that open new tabs and aren't on domains we ultimately manage ourselves. (Looks like the webinar page covers this for the text link; the image looks like it doesn't target a new window.) It's worth taking a quick pass to make sure these layouts handle this properly (as admittedly, this slipped under my radar as well).

# Event information
event:
    # The type of activities we will be doing at the event.
    type: ["talk"]
    # The event address
    location: "Online"
    # The start date of an event. Format YYYY-MM-DD
    start_date: "2020-10-07"
    # The end date of an event. Format YYYY-MM-DD
    end_date: "2020-10-08"
    # The event description shown on the event list page.

>     description: "This annual Cloud Engineering Summit brings together leaders and practitioners to share best practices and end-user stories on a broad range of cloud engineering topics."

Reviewer 2: ```suggestion
    description: "The Cloud Engineering Summit brings together leaders and practitioners to share best practices and end-user stories on a broad range of cloud engineering topics from architecture to reliability and security."
```
